### ISSUE TITLE
MiniHUD for Forge 1.16.5

### ISSUE THREAD

**Issue by MrYoavon** [maruohon/minihud#238]
Hey,
I am playing on a modded server with some friends and our modpack uses Forge to run and not Fabric.
The modpack is on version 1.16.5 but I can't find a MiniHUD version for Forge on that version (I can see 1.14 and 1.15 but not 1.16.5)

Could you please implement it into the game? It would be very helpful to us.

Thank's anyway.
MrYoavon

**Comment by maruohon** [comment]
Forge ports of all my client mods are planned, but I won't make them from the current old code base anymore.

See the longer explanation here: https://www.curseforge.com/minecraft/mc-mods/malilib-forge

**Comment by FeryFaithless** [comment]
Yes,PLEAAAASE. 1.16.5 FORGE of course .we really need thm.Dorood dear
